src/tests/multihost/alltests/test_misc_proxy.py

```diff
@@ -0,0 +1,391 @@
+        :id: ce5fe4d2-e6e0-11ec-af7d-845cf3eff344
+        :steps:
+            1. Configure SSSD chpass_provider = proxy
+            2. Run authconfig --test
+        :expectedresults:
+            1. Should succeed
+            2. Should succeed
+        """
+        # more than 10 auth attempt times out bz647816
+        tools = sssdTools(multihost.client[0])
+        domain_name = tools.get_domain_section_name()
+        domain_params = {'id_provider': 'ldap',
+                         'auth_provider': 'ldap',
+                         'chpass_provider': 'proxy',
+                         'proxy_pam_target': 'sssdproxyldap',
+                         'ldap_schema': 'rfc2307',
+                         'enumerate': 'false',
+                         'cache_credentials': 'true' }
+        tools.sssd_conf('domain/' + domain_name, domain_params)
+        tools.clear_sssd_cache()
+        execute_cmd(multihost, "authselect test sssd")
+
+    def test_bz1221992(self, multihost, backupsssdconf):
+        """
+        :title: sssd_be segfault at 0 ip sp error 6 in libtevent.so.0.9.21
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1221992
+        :id: a97bf86a-e6e3-11ec-a1ee-845cf3eff344
+        :steps:
+            1. Configure user=sssd  is in in sssd.conf
+            2. user should be able to change the
+             password without segfault
+        :expectedresults:
+            1. Should succeed
+            2. Should succeed
+        """
+        tools = sssdTools(multihost.client[0])
+        domain_name = 'sssd'
+        domain_params = {'user': 'sssd'}
+        tools.sssd_conf(domain_name, domain_params)
+        domain_name = tools.get_domain_section_name()
+        domain_params = {'id_provider': 'ldap',
+                         'auth_provider': 'ldap',
+                         'chpass_provider': 'proxy',
+                         'proxy_pam_target': 'sssdproxyldap',
+                         'proxy_lib_name': 'ldap' }
+        tools.sssd_conf('domain/' + domain_name, domain_params)
+        execute_cmd(multihost, "> /var/log/messages")
+        tools.clear_sssd_cache()
+        execute_cmd(multihost, "sh /tmp/sssdproxymisc.sh")
+        time.sleep(3)
+        with pytest.raises(subprocess.CalledProcessError):
+            execute_cmd(multihost, "grep 'segfault at 0 ip' /var/log/messages")
+        assert 'sssd' in execute_cmd(multihost,
+                                     "stat -c %G /var/lib/sss/pipes"
+                                     "/private/sbus-dp_example1.*").stdout_text
+
+    def test_0002_bz1209483(self, multihost, backupsssdconf):
+        """
+        :title: sssd does not work as expected when id provider
+         equal to proxy and auth provider equal to ldap bz1209483
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1209483
+        :id: 2fa4834c-b252-11ec-919d-845cf3eff344
+        :steps:
+          1. Adding the user in local
```

Comment:
Update this to  `Add a local user with the same name as an existing ldap user`

Reply:
Done